### PR TITLE
Newsletter May 2024

### PR DESCRIPTION
**Pull request by davidfischer**

### CODE REVIEW

File: content/posts/newsletter-may-2024.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+* We did a wider demo and tech explanation of our
+  [page similarity]({filename}../pages/similar-pages.md?url=https%3A%2F%2Fwww.mongodb.com%2Fatlas)
+  features that will help us target small audiences by matching pages where our ads appear directly
+  with advertiser landing pages.
+  The integration into our ad targeting is still being worked on and should be released later this month
+  or early June.
+* We released a UX improvement for advertisers to let them more easily and quickly
+  compare campaigns on a CTR and eCPC basis.
+* We had a content piece exploring the trade-offs publishers make and why
```
Comment (ericholscher) [inline content/posts/newsletter-may-2024.md:20]: Did we post about this on social?

Comment (davidfischer) [reply content/posts/newsletter-may-2024.md:20]: Looks like there was a draft in buffer but never sent. Want to give it a look?

Comment (ericholscher) [reply content/posts/newsletter-may-2024.md:20]: Looks great -- we should send it to LI as well.

Comment (davidfischer) [reply content/posts/newsletter-may-2024.md:20]: I did send a message in slack but probably needed to tag you. Then I forgot about it =(

**Pull request merged**